modify distributed transaction and ttl.md (#257)
tikv/website: content/docs/5.1/concepts/explore-tikv-features/ttl.md

> 

Author: Here an introduction on TTL is missing. I think we should first explain what is TTL and why we use TTL on RawKV (their relations). @marsishandsome Could you please give a brief introduction on TTL here? Then I can create another pull request to fix this problem.

Reviewer: TTL 指的是 Time To Live。
- 如果不用 TTL，那么往 TiKV 写入的数据，会一直存在在 TiKV 中，除非有人删除。
- 如果使用 TTL，例如设置某个 key 的 TTL = 1 小时，那边 1 小时后这个数据会被 TiKV 自动删除。

Author: This explanation has been added in https://github.com/tikv/website/pull/268

## Prerequisites

> Before you start, ensure that you install TiUP, jshell and download tikv-client jars according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).

Reviewer 2: ```suggestion
Before you start, ensure that you have installed TiUP and jshell, and have downloaded the `tikv-client` JARS file according to [TiKV in 5 Minutes](../../tikv-in-5-minutes).
```

## Step 1: Config TiKV to enable TTL

> TTL is disabled by default. You need to create a file `tikv.yaml` using the following statement to enable TTL.

Reviewer 2: ```suggestion
TTL is disabled by default. You need to create a file `tikv.yaml` using the following configuration to enable TTL.
```

## Step 2: Start TiKV Cluster

> For this tutorial, you only need one TiKV node, so use the `tiup playground` command.

Reviewer 2: ```suggestion
For this tutorial, only one TiKV node is needed, so the `tiup playground` command is used.
```
